components/bl-copyButton-component/src/index.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useMemo } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CopyButton({ component, eventHandlers }) {
+  const { display, classList, isDisabled, content, copyLabel, copiedLabel, copiedDuration } = component;
+  const { onCopy } = eventHandlers;
+
+  const [isCopied, setIsCopied] = useState(false);
+
+  component.copy = () => {
+    navigator.clipboard.writeText(content).then(() => {
```

Comment:
Do this solution works well on every popular mobile and desktop browsers?

Reply:
I think that yes
![image](https://user-images.githubusercontent.com/99363993/183071706-e3495601-d5b6-43b5-b2a4-880616a30777.png)